chapter-2-control-flow.md

```diff
@@ -37,6 +37,15 @@ Here is a simple for statement:
 </div>
 ```
 
+You can expose the index of the current iteration by following the syntax `{{for i, j in k}}`):
```

Comment:
would be more clear to say `{{for name, index in collection}}` instead of using i, j, as usually i and j are both indexes